ui/components/component-library/banner-tip/banner-tip.stories.tsx

```diff
@@ -1,32 +1,20 @@
 import React, { useState } from 'react';
+import { Meta, StoryFn } from '@storybook/react';
 import {
-  DISPLAY,
-  FLEX_DIRECTION,
-  Size,
+  Display,
+  FlexDirection,
 } from '../../../helpers/constants/design-system';
-import Box from '../../ui/box/box';
-import { ButtonLink, ButtonPrimary, Icon, IconName } from '..';
+import {
+  ButtonLink,
+  ButtonLinkSize,
+  ButtonPrimary,
+  Box,
+  Icon,
+  IconName,
+} from '..';
 import README from './README.mdx';
 import { BannerTip, BannerTipLogoType } from '.';
 
-const marginSizeControlOptions = [
-  undefined,
-  0,
-  1,
-  2,
-  3,
-  4,
-  5,
-  6,
-  7,
-  8,
-  9,
-  10,
-  11,
-  12,
-  'auto',
-];
-
```

Comment:
Removed as this component extends `BannerBase` which extends the `Box` component and hence all the controls are available